src/Rules/FunctionCallParametersCheck.php

```diff
@@ -196,6 +205,7 @@ public function check(
 				false,
 				$argumentName,
 				$arg->getStartLine(),
+				$nonUnpackAfterUnpacked,
```

Comment:
could be replaced with `$hasUnpackedArgument && !$arg->unpack` but not sure what's more readable